static/styles/settings.css

```diff
@@ -755,9 +755,9 @@ input[type=checkbox].inline-block {
 }
 
 #settings_page .content-wrapper {
-    position: relative;
-    float: left;
-    width: calc(100% - 250px - 2px);
+    position: absolute;
+    left: 251px;
+    width: calc(100% - 250px - 1px);
```

Comment:
Yeah, since you just changed the `left` value to 251px, I think this should have a comment.